TCAT/Cells/BusStopTableViewCell.swift

```diff
@@ -44,7 +44,7 @@ class BusStopTableViewCell: UITableViewCell {
         connectorBottom.backgroundColor = .tcatBlueColor
         contentView.addSubview(connectorBottom)
         
-        statusCircle = Circle(size: .small, color: .tcatBlueColor, style: .outline)
+        statusCircle = Circle(type: .smallOutline, color: .tcatBlueColor)
```

Comment:
why the change? :( i don't see the point of adding this new enum that encapsulates the size and style b/c it makes the Circle class less flexible. Can we go back??

Reply:
I'll revert, but different permutations of stuff are sorta untested